### ISSUE TITLE
search2 result list does not read my-collection reference

### ISSUE THREAD

**Issue by GVogeler** [icaruseu/mom-ca#476]
http://monasterium.net/mom/search2?q=K%C3%B6ln+Laurentius&option=and&img=&annotations=&sort=date&categories=&context= gives links to "Salzburg Landesarchiv ..." but not to "Illuminierte Urkunden" (user collection)

**Comment by GVogeler** [comment]
see #469

**Comment by StephanMa** [comment]
Is this issue still up?

![unbenannt](https://cloud.githubusercontent.com/assets/6735556/21840957/7f569922-d7e0-11e6-8198-f8ee96e4c26e.JPG)

**Comment by GVogeler** [comment]
yes: before `(collection)` in other collections the name and a link to the collection is shown, e.g.: http://monasterium.net/mom/search?q=%22Kaiser+Heinrich+III.%22&sort=date&arch=&col=ArdCan
![grafik](https://cloud.githubusercontent.com/assets/324006/21844630/b709ca92-d7ef-11e6-9d83-6aa07fde8cdb.png)

**Comment by GVogeler** [comment]
probably due to introduction of eXist-collection `metadata.my-collection.public` long after implementing search results, so https://github.com/icaruseu/mom-ca/blob/d60ccd85534fad70b9ae7995c5fa58f571849fa9/my/XRX/src/mom/app/search/widget/search-result.widget.xml#L92 might be the place to start?

**Comment by StephanMa** [comment]
Okay... i would call it <i>data inconsistency</i>...
Search2 does display the `cei:provenance/child::text()` as text for the collection. 
Search is using `cei:title/text()` ...

So we have to decided which one we have to choose and if done, add the missing data.

**Comment by maburg** [comment]
Both searches find mycollection hits! everything seems alright.
